src/main.zig

```diff
@@ -2356,6 +2357,8 @@ fn buildOutputType(
                         fatal("unable to parse /version '{s}': {s}", .{ arg, @errorName(err) });
                     };
                     have_version = true;
+                } else if (mem.eql(u8, arg, "--no-undefined-version")) {
+                    linker_no_undefined_version = true;
```

Comment:
Also add handling for `--undefined-version`:

```
                } else if (mem.eql(u8, arg, "--undefined-version")) {
                    linker_no_undefined_version = false;
```

Reply:
doing this with a seperate arg to make it a little more explicit

Reply:
> doing this with a seperate arg to make it a little more explicit

The surface area in the API should be kept as small as we can make it. The no-`no-` variant should be kept in front-end as per later suggestion.